src/main/scala/temple/DSL/semantics/Validator.scala

```diff
@@ -1,44 +1,81 @@
 package temple.DSL.semantics
 
+import temple.DSL.semantics.NameClashes._
+import temple.DSL.semantics.Validator._
 import temple.ast.AttributeType._
 import temple.ast.{Metadata, _}
+import temple.builder.project.ProjectConfig
 
 import scala.collection.mutable
 import scala.reflect.{ClassTag, classTag}
 
-private class Validator private (templefile: Templefile) {
-  var errors: mutable.Set[String] = mutable.SortedSet()
+private class Validator(templefile: Templefile) {
+  var errors: mutable.Buffer[String] = mutable.Buffer()
```

Comment:
Is the mutable prefix needed? Don't think it'd make much sense to have an `immutable.Buffer` 😆

Reply:
Good point, was a mutable set so I’ll fix that